frontend/packages/console-shared/src/components/SvgPodTooltip.tsx

```diff
@@ -1,9 +1,8 @@
 import * as React from 'react';
-import SvgDropShadowFilter from '../svg/SvgDropShadowFilter';
-import { createSvgIdUrl } from '../../utils/svg-utils';
-import { podColor } from './topology-utils';
-
+import SvgDropShadowFilter from '../../../dev-console/src/components/svg/SvgDropShadowFilter';
```

Comment:
Did you try to use the PF4 tooltip instead? If so we could remove this and save a whole lot of refactoring.

Reply:
Yes, I tried to use PF4 tooltip. But didn't works in our case. Because for PF4 I have to wrap the ```VictoryPie``` with ```Tooltip``` component and It doesn't render the graph properly.

Reply:
Ok. Let's move to using the PF4 chart tooltips then.